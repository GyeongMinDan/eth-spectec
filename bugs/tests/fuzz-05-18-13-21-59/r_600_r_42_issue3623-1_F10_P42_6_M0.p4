enum bit<4> e {
 a = ((string) (true)) };
