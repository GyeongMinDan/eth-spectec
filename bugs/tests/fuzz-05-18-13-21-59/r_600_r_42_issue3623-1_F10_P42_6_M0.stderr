likely_hits/r_600_r_42_issue3623-1_F10_P42_6_M0.p4(2): [--Werror=type-error] error: string: cast not supported
 a = ((string) (true)) };
       ^^^^^^
In file: /p4c/frontends/p4/typeChecking/typeChecker.cpp:172
Compiler Bug: /p4c/frontends/p4/typeChecking/typeChecker.cpp:172: Null srcType

